index.html

```diff
@@ -42,6 +42,148 @@ <h3>I am a software developer</h3>
                    
                 </ul>
                 
+        </section>
+                <!-- work section -->
+        <section> 
+            <div class="work-section-container">
+                <h2> My Recent Works </h2>
+                <hr>
+                
+                <div class="work-container">
+                    <div class="item work-item-1">
+                        <img src="image/Img_Placeholder.svg" alt="works">
```

Comment:
- [ ] Following the design and layout of Figma is a requirement. Kindly make sure that the spacing between the previous section and the work section is pretty much the same as that of Figma. And also make sure that the spacing between the header **My Recent Works** and the image is the same as that of Figma. It will be nice if you make the text **My Recent Works** horizontal and get rid of the borders. Let's check out the screenshots below in case of any doubts.


Your Implementation | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/164865968-f0eaac22-17db-42f2-b5fc-9a49f9dc3d10.png) | ![](https://user-images.githubusercontent.com/63087491/164865972-3ff4bf31-c029-4e37-b016-b2df40656b1a.png)